dotnet/src/Connectors/Connectors.AI.Oobabooga/TextCompletion/TextCompletionStreamingResponse.cs

```diff
@@ -5,11 +5,28 @@
 namespace Microsoft.SemanticKernel.Connectors.AI.Oobabooga.TextCompletion;
 
 /// <summary>
-/// HTTP Schema for streaming completion response. Adapted from <see href="https://github.com/oobabooga/text-generation-webui/blob/main/extensions/api/streaming_api.py"/>
+/// Represents the HTTP schema for streaming completion response. Adapted from <see href="https://github.com/oobabooga/text-generation-webui/blob/main/extensions/api/streaming_api.py"/>.
+/// <example>
+/// <code>
+/// var response = new TextCompletionStreamingResponse
+/// {
+///     Event = TextCompletionStreamingResponse.ResponseObjectTextStreamEvent,
+///     MessageNum = 0,
+///     Text = "Generated text"
+/// };
+/// </code>
+/// </example>
 /// </summary>
 public sealed class TextCompletionStreamingResponse
 {
+    /// <summary>
+    /// Constant string representing the "text_stream" event in the websocket message.
```

Comment:
Nit: This isn't clear. Should it be?

`Constant string representing the event that is fired when text is received from a websocket.`